src/StatusQ/Core/Utils/Emoji.qml

```diff
@@ -19,7 +19,7 @@ QtObject {
         "svg": "svg"
     }
     property string base: Qt.resolvedUrl("../../../assets/twemoji/")
-
+    property var emojiJSON: EmojiJSON
```

Comment:
Probably a stupid questoin: Where does `EmojiJSON` come from now? I guess the answer lies in it being a singleton instance? Is this a breaking change?

Reply:
@PascalPrecht EmojiJSON is imported above `import "./emojiList.js" as EmojiJSON` here we just create an instance of it so that it can be referenced in the EmojiPopup, see https://github.com/status-im/status-desktop/pull/6745/files#diff-298b4f701bf6ed9246dd67854beec455eb9c421be43554248043dea78c15da39

Reply:
Ah thanks for clarifying!
Just to be clear though: without the change in Status Desktop, that will break?

Reply:
@PascalPrecht as far as I can see, this is only used in `StatusEmojiPopup.qml`
<img width="477" alt="em" src="https://user-images.githubusercontent.com/31625338/182394541-fda24acc-066e-4375-90a6-13536b429ee6.png">

Reply:
Cool, thanks for veriyfing!